README.md

```diff
@@ -75,7 +70,7 @@ You can go check GitHub webhook delivery payloads and responses on the [CLA Bot'
 App Advanced
 Settings](https://github.com/organizations/adobe/settings/apps/adobe-cla-bot/advanced).
 
-#### Check logs on Adobe Runtime
+#### Check Logs on Adobe Runtime
 
 Get list of actions running on our namespace:
```

Comment:
`wsk activation get <activation_id> is for getting list of invocations. We should change the wording.